src/Dispatcher.php

```diff
@@ -6,27 +6,27 @@
 use Exception;
 use ReflectionClass;
 use FastRoute\Dispatcher as BaseDispatcher;
-use FastRoute\Dispatcher as FastRouterDispatcherInterface;
```

Comment:
sadly the class name is already `Dispatcher` we have to let as `BaseDispatcher` or another name like: `DispatcherInterface` if you prefere